GANDLF/utils/gandlf_logging.py

```diff
@@ -26,6 +28,16 @@ def _configure_logging_with_logfile(log_file, config_path):
         logging.config.dictConfig(config_dict)
 
 
+def gandlf_excepthook(exctype, value, tb):
+    if issubclass(exctype, AssertionError):
+        logging.exception(
+            f"{exctype.__name__}: {value}\n{''.join(traceback.format_exception(exctype, value, tb))}",
+            exc_info=False,
+        )
```

Comment:
```suggestion
        logging.exception("Uncaught exception", exc_info=(exctype, value, tb))
```
`Logging` can pretty-print exceptions on its own. Ow do we need a special formatting?